test.ts

```diff
@@ -158,6 +158,20 @@ describe('IGCParser', () => {
       additionalData: null,
     });
 
+    test.pass('AXCT XCTrack 0.6.0 on samsung GT-S6500 2.3.6', {
+      manufacturer: 'XCT',
+      loggerId: ' XCTrack 0.6.0 on samsung GT-S6500 2.3.6',
```

Comment:
I think I would prefer it if this string was in `additionalData` instead of the `loggerId` 🤔 

The `loggerId` field has a defined format and this is certainly not complying with that format.

Can you check if adjusting the regex with `(?:FLIGHT:(\d+)|\:(.+)| (.+))` solves the issue?

Reply:
According to the spec:

> A MMM NNN: TEXT STRING CR LF
> The FR Serial ID, valid characters alphanumeric. For IGC FRs this must be three alphanumeric characters. Non-IGC FRs may use different systems. 

Additional would appear after `:` so I guess we are in the "Non-IGC FRs may use different systems." here ?

What do you think ?

Reply:
maybe, but "XCTrack 0.6.0 on samsung GT-S6500 2.3.6" doesn't look like a `loggerId` to me. it looks more like a version string IMHO.

Reply:
I chose to populate `loggerId` because:
1. It seems to be compliant with the spec (as quoted in my previous message),
2. There must be a `loggerId` for the result to be valid, see https://github.com/Turbo87/igc-parser/blob/master/index.ts#L181-L183

Also I have found the following A record generated by a Syride, SYS'Nav: `AXSR`

So open questions are:
1. Should we move non-standard ids to `additionalData` ?
2. Should we check the manufacturer instead of the `loggerId` for the A field as you mentioned in the other PR ?

Reply:
> Should we check the manufacturer instead of the loggerId for the A field as you mentioned in the other PR ?

yes, I think that makes sense

> Should we move non-standard ids to additionalData ?

👍